dev/NavigationView/NavigationView.cpp

```diff
@@ -1115,6 +1119,51 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    if (PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::Top
+        || DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+    {
+        if (auto backButton = m_backButton.try_as<winrt::Button>())
+        {
+            backButton.Width(40);
+        }
+        if (auto paneToggleButton = m_paneToggleButton.try_as<winrt::Button>())
+        {
+            paneToggleButton.MinWidth(40);
+        }
+        if(auto paneButton = m_paneToggleButton.try_as<winrt::Button>())
+        {
+            if (auto paneToggleButtonIconColumn = paneButton.GetTemplateChild(c_paneToggleButtonIconGridColumnName).try_as<winrt::ColumnDefinition>())
```

Comment:
>GetTemplateChild [](start = 61, length = 16)

If this GetTemplateChild fails and returns null I think the try_as call would crash.

Reply:
So should we wrap this in another if case? The problem why I do not have a "static" reference here is because by the time OnApplyTemplate() gets called, the column does not exist yet.

Reply:
yeah, spliting into two if statments is probably needed

Reply:
Fixed